src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -91,11 +91,11 @@ class ItemPage extends React.Component {
           definition: (
             <ul>{record[f.field].map((obj, i) => (
               <li key={i}>
-                <a
+                <Link
                   onClick={e => this.onClick(e, `${f.field}:"${obj['@id']}"`)}
                   title={`Make a new search for ${f.label}: ${obj.prefLabel}`}
-                  href={`/search?q=${encodeURIComponent(`${f.field}:"${obj['@id']}"`)}`}
-                >{obj.prefLabel}</a>
+                  to={`/search?q=${encodeURIComponent(`${f.field}:"${obj['@id']}"`)}`}
```

Comment:
These links need to be updated per #337 . Can you work that into this PR since it's related?